orttraining/orttraining/test/python/orttraining_run_multiple_choice.py

```diff
@@ -0,0 +1,241 @@
+        expected_loss = 0.6469693916158167
+
+        results = self.run_multiple_choice(model_name="bert-base-cased", task_name="swag", fp16=True)
+        assert_allclose(results['acc'], expected_acc)
+        assert_allclose(results['loss'], expected_loss)
+
+    def run_multiple_choice(self, model_name, task_name, fp16):
+        model_args = ModelArguments(model_name_or_path=model_name, cache_dir=self.cache_dir)
+        data_args = DataTrainingArguments(task_name=task_name, data_dir=self.data_dir,
+            max_seq_length=self.max_seq_length)
+
+        training_args = TrainingArguments(output_dir=self.output_dir + "/" + task_name, do_train=True, do_eval=True,
+            per_gpu_train_batch_size=self.train_batch_size,
+            per_gpu_eval_batch_size=self.eval_batch_size,
+            learning_rate=self.learning_rate, num_train_epochs=self.num_train_epochs,local_rank=self.local_rank,
+            overwrite_output_dir=self.overwrite_output_dir, gradient_accumulation_steps=self.gradient_accumulation_steps,
+            fp16=fp16, logging_steps=self.logging_steps)
+
+        # Setup logging
+        logging.basicConfig(
+            format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
+            datefmt="%m/%d/%Y %H:%M:%S",
+            level=logging.INFO if training_args.local_rank in [-1, 0] else logging.WARN,
+        )
+        logger.warning(
+            "Process rank: %s, device: %s, n_gpu: %s, distributed training: %s, 16-bits training: %s",
+            training_args.local_rank,
+            training_args.device,
+            training_args.n_gpu,
+            bool(training_args.local_rank != -1),
+            training_args.fp16,
+        )
+        logger.info("Training/evaluation parameters %s", training_args)
+
+        set_seed(training_args.seed)
+        onnxruntime.set_seed(training_args.seed)
+
+        try:
+            processor = processors[data_args.task_name]()
+            label_list = processor.get_labels()
+            num_labels = len(label_list)
+        except KeyError:
+            raise ValueError("Task not found: %s" % (data_args.task_name))
+
+        config = AutoConfig.from_pretrained(
+            model_args.config_name if model_args.config_name else model_args.model_name_or_path,
+            num_labels=num_labels,
+            finetuning_task=data_args.task_name,
+            cache_dir=model_args.cache_dir,
+        )
+        tokenizer = AutoTokenizer.from_pretrained(
+            model_args.tokenizer_name if model_args.tokenizer_name else model_args.model_name_or_path,
+            cache_dir=model_args.cache_dir,
+        )
+
+        model = AutoModelForMultipleChoice.from_pretrained(
+            model_args.model_name_or_path,
+            from_tf=bool(".ckpt" in model_args.model_name_or_path),
+            config=config,
+            cache_dir=model_args.cache_dir,
+        )
+
+        # Get datasets
+        train_dataset = (
+            MultipleChoiceDataset(
```

Comment:
Will it throw a friendly exception in case the dataset is not found? I imagine that devs running this locally may not have the dataset, so a meaningful error message informing that would help.

Reply:
exception with meaningful error is throw if data file does not exist.